client/organizations.go

```diff
@@ -58,9 +59,20 @@ func loadAccounts(ctx context.Context, awsConfig *Config, accountsApi Organizati
 		if account.Status != orgTypes.AccountStatusActive {
 			continue
 		}
+		roleArn := arn.ARN{
+			Partition: "aws",
```

Comment:
I think the partition can't be hardcoded

Reply:
we parse and replace it below.